### ISSUE TITLE
npm install not working because of event-stream@3.3.6

### ISSUE THREAD

**Issue by nicu-chiciuc** [videojs/generator-videojs-plugin#266]
I created a new folder, ran `yo videojs-plugin --skip-install` then `npm install`. I'm getting this error:
```
npm ERR! code E404
npm ERR! 404 Not Found: event-stream@3.3.6
```
This is because that version of the package was removed from npm because of the incident that happened in November: (https://blog.npmjs.org/post/180565383195/details-about-the-event-stream-incident)

I think this should definitely fixed ASAP

**Comment by nicu-chiciuc** [comment]
So the bad dependencies were these:
`npm-run-all` -> `ps-tree`-> `event-stream`
This was resolved here: https://github.com/mysticatea/npm-run-all/issues/149
I've updated `npm-run-all` from `4.1.3` to `4.1.5` and the install went through (although there are 2 high vulnerabilities, but this is another issue).

**Comment by misteroneill** [comment]
Fixed by #267
